netbox/extras/filtersets.py

```diff
@@ -326,6 +327,59 @@ def _usable(self, queryset, name, value):
         return queryset.filter(Q(enabled=False) | Q(Q(shared=False) & ~Q(user=user)))
 
 
+class TableConfigFilterSet(ChangeLoggedModelFilterSet):
+    q = django_filters.CharFilter(
+        method='search',
+        label=_('Search'),
+    )
+    object_type_id = django_filters.ModelMultipleChoiceFilter(
+        queryset=ObjectType.objects.all(),
+        field_name='object_type'
+    )
+    object_type = ContentTypeFilter(
+        field_name='object_type'
+    )
```

Comment:
This one doesn't seem to be working. At this point, all of my existing saved configs are for object type dcim.Device. But, as you can see, none are returned when filtering for that object type.

![image](https://github.com/user-attachments/assets/51549553-a7e2-45b9-997a-095628bfa61c)

Reply:
Derp, this belongs on the filter form below not here, my bad.